debezium-parent/pom.xml

```diff
@@ -207,6 +208,7 @@
                     <systemPropertyVariables combine.children="append">
                         <java.io.tmpdir>${project.build.directory}</java.io.tmpdir>
                         <skipLongRunningTests>${skipLongRunningTests}</skipLongRunningTests>
+                        <failFlakyTests>${failFlakyTests}</failFlakyTests>
```

Comment:
@Naros Should not be failsafe plugin configured too if we want to make it cetralized?